packages/react/package.json

```diff
@@ -89,6 +89,7 @@
     "@storybook/manager-webpack5": "^6.5.6",
     "@storybook/react": "^6.5.6",
     "@storybook/theming": "^6.5.6",
+    "@types/react-is": "^17.0.3",
```

Comment:
Instead of having this listed in devDependencies, I think this can instead be placed under `resolutions` in the root `package.json` alongside the types packages for react and prop-types

Reply:
`resolutions` are, AFAIK, for version enforcing… right now, the `types` package wasn't there at all & I've been led to believe that dev-dependencies is the place for `@types/` packages since that is what yarn's typescript plugin does, and it makes an intuitive sense to me… that said I will, of course, concur to what is your way of doing this.

Reply:
So, I have just tried what you proposed, but as expected, that doesn't work… resolutions only force versions, they don't add dependency if it's not already declared somewhere